app/Http/Controllers/ActionStatsController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Rogue\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Rogue\Models\ActionStat;
+use Rogue\Http\Transformers\ActionStatTransformer;
+
+class ActionStatsController extends ApiController
+{
+    /**
+     * @var Rogue\Http\Transformers\ActionTransformer;
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     */
+    public function __construct()
+    {
+        $this->transformer = new ActionStatTransformer;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $query = $this->newQuery(ActionStat::class);
+
+        $filters = $request->query('filter');
+        $query = $this->filter($query, $filters, ActionStat::$indexes);
+
+        return $this->paginatedCollection($query, $request);
+    }
```

Comment:
Would adding a "show" endpoint here make sense (rather than filtering the index down to an array of one)?

Reply:
Going to leave this as just an `index` endpoint per https://github.com/DoSomething/rogue/pull/964/files#discussion_r353974591